Tests/MySQLTests/MySQLMigrationTests.swift

```diff
@@ -0,0 +1,75 @@
+import Async
+@testable import MySQL
+import SQLBenchmark
+import XCTest
+
+class MySQLMigrationTests: XCTestCase {
```

Comment:
I'd prefer to have these in `MySQLTests.swift`. That file already tests `MySQLQuery` so it would make sense for these tests to also be there.